Database/Corrections/sodQuestFixes.lua

```diff
@@ -239,6 +239,11 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -425170,
             [questKeys.zoneOrSort] = sortKeys.MAGE,
         },
+        [78144] = {
+            [questKeys.questLevel] = -1,
+            [questKeys.requiredSpell] = -425444,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
```

Comment:
```suggestion
            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
            [questKeys.preQuestGroup] = {78132,78133,78134},
```

All three quests are required before you get this escort quest.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5416/commits/b251e6416f0fb61b5e42593546b490594eba3c7b